tuxemon/technique.py

```diff
@@ -214,7 +215,11 @@ def full_recharge(self) -> None:
     def reset_combat_counter(self) -> None:
         """Reset the combat counter."""
         self._combat_counter = 0
-
+    def keep_old_stats(self):
+        for player in self.active_players:
```

Comment:
technically, there could be more than oner player, or players could leave.  maybe not now, but in the future, maybe.  this is going to cause problems.  old_stats should keep a reference to the monster.  this should be a dictionary